app/Fans.cs

```diff
@@ -450,6 +457,75 @@ private void ChartCPU_MouseMove(object? sender, MouseEventArgs e)
 
 
         }
+
+        private void AdjustAllLevels(int index, double curYVal,  double curXVal, Series series) {
+            
+            // Get the neighboring DataPoints of the hit point
+            DataPoint upperPoint = null;
+            DataPoint lowerPoint = null;
+
+            if (index > 0) 
+            {
+                lowerPoint = series.Points[index - 1];
+            }
+
+            if (index < series.Points.Count)
```

Comment:
Probably going to break the graph tooltips because of out of bounds exception when interacting with the last node in the graph.